PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -531,6 +550,20 @@ def getCorrectedMET(self, process, metType, correctionLevel,produceIntermediateC
         #Enable MET significance if the type1 MET is computed
         if "T1" in correctionLevel:
             getattr(process, "pat"+metType+"Met"+postfix).computeMETSignificance = cms.bool(True)
+            getattr(process, "pat"+metType+"Met"+postfix).srcPFCands =  cms.InputTag("packedPFCandidates")
+            if self._parameters["runOnData"].value:
+                getattr(process, "pat"+metType+"Met"+postfix).parameters = METSignificanceParams_Data
```

Comment:
I thought that miniAOD running on data actually has this parameter "runOnData" set.
It looks like it is not the case.

If it matters, you may want to follow up (in a separate PR?) and percolate the settings appropriately from miniAOD_customizeAllData called by the cmsDriver's configs for data.